pulp_rpm/app/constants.py

```diff
@@ -1,26 +1,17 @@
 from types import SimpleNamespace
 
+from pulpcore.plugin.constants import ALL_KNOWN_CONTENT_CHECKSUMS
+
+# Due to unsorted 'ALL_KNOWN_CONTENT_CHECKSUMS' must be sorted
+# when used or Django would like to create new migration
+
+
 CHECKSUM_TYPES = SimpleNamespace(
-    UNKNOWN="unknown",
-    MD5="md5",
-    SHA="sha1",  # compatibility nickname from original createrepo
```

Comment:
What do we do with this compatibility aspect?
If I understand correctly, a checksum can be specified as `sha` in some old repos. Correct?

Reply:
Those changes was removed as out of scope this issue.